packages/framework/src/Resources/styles/admin/core/form/custom-inputs.less

```diff
@@ -0,0 +1,114 @@
+@custom-inputs-margin-left: 30px;

```

Comment:
This file is indented by `TAB`s instead of spaces and uses `\r\n` instead of `\n` for line ending. Did you copy it from a different project, or do you have a misconfigured IDE?

Reply:
Yes, it is from another project so I fixed it.